examples/notebooks/show_remote_cubes.py

```diff
@@ -0,0 +1,34 @@
+import os
+import pandas as pd
+import s3fs
+from xcube.api import read_cube
+
+def show_remote_cubes():
```

Comment:
- Add options allowing users to use this function for other buckets and endpoints.
- Could/should be integral part of xcube?